retux/client/mixins/editable.py

```diff
@@ -0,0 +1,72 @@
+from ..bot import Bot
+from ...api.http import _Route, _RouteMethod
+from attrs import asdict
+
+__all__ = ["Editable"]
+
+
+class Editable:
+    """
+    A mixin for objects with edit/delete methods.
+
+    Methods
+    _______
+    `edit()` : `dict`
+        Edits an object with the given path and kwargs.
+    `modify()` : `dict`
+        An alias of `edit()`
+    `delete()` : `dict | None`
+        Delete an object from discord.
+    """
+
+    async def edit(self, bot: Bot, path: str, **kwargs) -> dict:
+        """
```

Comment:
This method is documented in the class' docstring but lacks the documentation in the method itself.

```suggestion
        """
        Edits an object with the given path and kwargs.

```